src/NuGetGallery.Core/GitHub/GitHubCacheTransformer.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NuGetGallery.GitHub
+{
+    public class GitHubCacheTransformer
+    {
+        public static Dictionary<string, NuGetPackageGitHubInformation> GetNuGetPackagesDependents(IReadOnlyList<RepositoryInformation> repositories)
```

Comment:
Can this be moved to `GitHubUsageConfiguration`? This makes testing a tad harder, but will hide implementation details a little better.